source/github-helpers/index.ts

```diff
@@ -17,7 +17,7 @@ export const getConversationNumber = (): string | undefined => {
 };
 
 const typesWithCommittish = new Set(['tree', 'blob', 'blame', 'edit', 'commit', 'commits', 'compare']);
-const titleWithCommittish = / at (?<branch>[\w-/]+)( · [\w-]+\/[\w-]+)?$/i;
+const titleWithCommittish = / at (?<branch>[\w-./]+)( · [\w-]+\/[\w-]+)?$/i;
```

Comment:
I don't remember if `\w-/` matches "from \w to /", so let's move this earlier

```suggestion
const titleWithCommittish = / at (?<branch>[.\w-/]+)( · [\w-]+\/[\w-]+)?$/i;
```

Reply:
@fregante ooops, It was typo. It must be `\.`. I expected allowing only character `.`.